frame/staking/src/tests.rs

```diff
@@ -3522,37 +3531,38 @@ fn two_different_bond_then_unbond_specific_one() {
 			36,
 		));
 		// TODO: kton free balance not correct
-		// assert_eq!(Kton::free_balance(&stash), 4);
+		// @clearloop: Fixed.
+		assert_eq!(Kton::free_balance(&stash), 4);
 
-		// // Bond 1 Kton
-		// assert_ok!(Staking::bond_extra(
-		// 	Origin::signed(stash),
-		// 	StakingBalance::KtonBalance(1),
-		// 	36
-		// ));
-		// assert_eq!(Staking::ledger(controller).unwrap().active_kton, 1);
+		// Bond 1 Kton
+		assert_ok!(Staking::bond_extra(
+			Origin::signed(stash),
+			StakingBalance::KtonBalance(1),
+			36
+		));
+		assert_eq!(Staking::ledger(controller).unwrap().active_kton, 1);
```

Comment:
## TODOS, pt1

TODOS in pt1 actually auto-fixed by deleting the comments, but I'm not sure about this function.